### PR TITLE
Project for setting up linters.

### PR DESCRIPTION
**Pull request by direakanbi**
# Project Descriptions
1. Created index.html
2. Added header.
3. Created styles.css
4. Modified styles.css
5. Pushed to feature branch.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @direakanbi, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+📝 License
+This project is MIT licensed.
```
Comment (mke2111) [inline README.md]: - [ ] In the readme file, add the necessary steps to follow to get started locally. How will someone without computer and coding proficiency be able to use install and use your app/repo

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Hello-Microverse</title>
+</head>
+<body>
+  <h1>Hello-Microverse!</h1>
```
Comment (mke2111) [inline index.html]: - [ ] Add a class to this heading, it is this class that you will use to apply your styles to within the CSS file.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```
Comment (mke2111) [inline styles.css]: - [ ] Apply your styles by using the class your will create for your heading element

**Review by ignatius22 — APPROVED** [review]
## Hi @direakanbi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**